WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFullScreenDialogFragment.java

```diff
@@ -130,4 +136,11 @@ public void onTaskTapped(QuickStartTask task) {
                 break;
         }
     }
+
+    @Override public void onSaveInstanceState(@NonNull Bundle outState) {
```

Comment:
```suggestion
    @Override
    public void onSaveInstanceState(@NonNull Bundle outState) {
```

Can we move the `@Override` annotation to a separate line than the method declaration for code style consistency?